react/OrderContext.tsx

```diff
@@ -0,0 +1,3 @@
+import { useOrder, OrderContext } from './components/OrderContext'
+
+export default { useOrder, OrderContext }
```

Comment:
Do we really need to expose `OrderContext`?

```suggestion
export default { useOrder }
```